e2e/test/scenarios/sharing/public-dashboard.cy.spec.js

```diff
@@ -181,4 +181,32 @@ describe("scenarios > public > dashboard", () => {
     cy.button("Apply").should("not.exist");
     cy.get(".ScalarValue").should("have.text", COUNT_DOOHICKEY);
   });
+
+  it("should respect dashboard width setting in a public dashboard", () => {
+    cy.get("@dashboardId").then(id => {
+      visitPublicDashboard(id);
+    });
+
+    // new dashboards should default to 'fixed' width
+    cy.findByTestId("fixed-width-container").should(
+      "have.css",
+      "width",
+      "1048px",
+    );
+
+    // toggle full-width
+    cy.get("@dashboardId").then(id => {
+      cy.signInAsAdmin();
+      cy.request("PUT", `/api/dashboard/${id}`, {
+        width: "full",
+      });
+      visitPublicDashboard(id);
+    });
```

Comment:
Do we actully need `cy.signInAsAdmin()` here?

Reply:
It was failing the request to change the width, so I had to add that in there